Controller/APIController.php

```diff
@@ -270,11 +270,12 @@ protected function getHeader(Request $request, $headerName)
      *
      * @return array
      */
-    protected function validateHeaders(array $headers, $required = true)
+    protected function validateHeaders(array $headers, $required = true, array $headerValidations = null)
```

Comment:
Variable name should not be in plural? $headerSValidations, since its an array?
Also, any specific reason to not set it as empty array instead of null?

Reply:
No reason for null, fixed now to an empty array.